distrho/src/DistrhoPluginAU.cpp

```diff
@@ -87,31 +84,26 @@ class PluginAU : public AUEffectBase
     {
         DISTRHO_SAFE_ASSERT_RETURN(inScope == kAudioUnitScope_Global, kAudioUnitErr_InvalidParameter);
 
+        outParameterInfo.flags = kAudioUnitParameterFlag_IsReadable | kAudioUnitParameterFlag_IsWritable;
```

Comment:
this is actually needed? I got my parameters to appear (and change values) in renoise with just the writable flag.
did you check what happens with parameter outputs?

Reply:
Just copied a template plugin.  I think it needs to be both readable and writable so you can change the param.